### ISSUE TITLE
Setting up a password to use the autogen module

### ISSUE THREAD

**Issue by Etiene** [Etiene/sailor#38]

**Comment by rnikhil275** [comment]
This seems like something I can try my hands on. Where do you want the user to enter the password ? When they go to ?r=autogen(after enabling it) do you want a form asking for the password ?

**Comment by Etiene** [comment]
Yes, something like that! The password could be set on the config below the enable
